.github/workflows/run-unit-tests.yml

```diff
@@ -67,6 +72,11 @@ on:
         required: false
         default: ''
         description: "Additional 'swift build' flags to be applied on when compiling with MUSL."
+      ios_xcodebuild_flag:
```

Comment:
I'd rather this be called `action` than `flag.`

Reply:
Done